src/Salt/SiteBundle/Entity/Comment.php

```diff
@@ -0,0 +1,367 @@
+<?php
+
+namespace Salt\SiteBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use JMS\Serializer\Annotation as Serializer;
+
+/**
+ * Comment
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="salt_comment")
+ */
+class Comment
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $parent;
```

Comment:
Might want to change this to be a `ManyToOne` referencing a `Comment` instead of just an int.